resources/views/includes/follow-button.blade.php

```diff
@@ -0,0 +1,21 @@
+@if($user->id == Auth::user()->id)
```

Comment:
When I request a person then after clicking the follow button the text says that you "no longer want to follow". When I reload the page, it says "Pending" correctly. Can you still adjust that one sees directly with the first click?

![image](https://user-images.githubusercontent.com/4103693/117777831-47a9ed80-b23d-11eb-8e79-55d9cfbeedcb.png)


![image](https://user-images.githubusercontent.com/4103693/117777849-4e386500-b23d-11eb-9384-aa8f443e3ad6.png)


![image](https://user-images.githubusercontent.com/4103693/117777874-555f7300-b23d-11eb-8656-9b966c530261.png)

Reply:
![photo](https://media.giphy.com/media/12WhqsC0MEe6IM/giphy.gif)